robot/physics.py

```diff
@@ -0,0 +1,69 @@
+#
+# See the notes for the other physics sample
+#
+
+
+from pyfrc.physics import motor_cfgs, tankmodel
+from pyfrc.physics.units import units
+import math
+
+
+class PhysicsEngine(object):
+    '''
+       Simulates a 4-wheel robot using Tank Drive joystick control
+    '''
+
+
+    def __init__(self, physics_controller):
+        '''
+            :param physics_controller: `pyfrc.physics.core.Physics` object
+                                       to communicate simulation effects to
+        '''
+
+        self.physics_controller = physics_controller
+
+        # Change these parameters to fit your robot!
+
+        self.drivetrain = tankmodel.TankModel.theory(
+            motor_cfgs.MOTOR_CFG_CIM,           # motor configuration
+            110*units.lbs,                      # robot mass
+            12.75,                              # drivetrain gear ratio
+            2,                                  # motors per side
+            24*units.inch,                      # robot wheelbase
+            25*units.inch,     # robot width
+            30*units.inch,     # robot length
+            6*units.inch                        # wheel diameter
+        )
+
+        self.kEncoder = 360 / (0.5 * math.pi)
```

Comment:
This value seems mighty mighty low.

Reply:
Those are some SMALL wheels.

Reply:
I think they're actually 6 inches, we can check to be sure @AndrewLester

Reply:
Maybe a good idea to make this a constant at any rate.

Reply:
It's in feet. I think I just kept it that way for consistency, but I'm not sure. The 360 is low though, as I think it's the encoder counts per revolution which is normally higher.

Reply:
that makes more sense, but then the encoder value still seems low I think